lib/Github/Api/Organization/Members.php

```diff
@@ -58,9 +58,13 @@ public function conceal($organization, $username)
     /*
      * Add user to organization
      */
-    public function add($organization, $username)
+    public function add($organization, $username, array $params = [])
     {
-        return $this->put('/orgs/'.rawurlencode($organization).'/memberships/'.rawurlencode($username));
+        if (isset($params['role']) && !in_array($params['role'], ['admin', 'member'])) {
+            $params['role'] = 'member';
+        }
```

Comment:
Can you remove this check and default value? We just pass the values to the github api and let them to the validation or set the default value for an option, this way our code is not incorrect or outdated if github changes something to their parameters.

Reply:
I tried to keep it similar to other code in the library:

https://github.com/KnpLabs/php-github-api/blob/eb4e3c2ef738386cac0c7d96e486f267a8f13eda/lib/Github/Api/Organization/Teams.php#L28-L30

If you still think these lines should be removed, please ping, I will gladly take care of it 👍

Reply:
Yes you are right that we do these kind of checks in other places of the api, but these are checks from the past and will be removed in the future (most likely 4.0, see #566)

So for new code we don't add those checks and purely rely on the github api to perform those checks!

Reply:
That makes sense, it will reduce the burden of maintenance. Let me patch this real quick 👍